search/search-dev-tools/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.0",
   "license": "GPLv2",
   "scripts": {
-    "build": "preact build --no-prerender --no-inline-css --no-sw",
+    "build": "preact build --no-prerender --no-inline-css --no-sw --no-esm",
```

Comment:
`preact-cli` now builds the ESM bundle by default; I have disabled it because we don't use it anyway.